### PR TITLE
Add start volume option

### PR DESCRIPTION
**Pull request by justincrosby**
https://github.com/fondberg/spotcast/issues/135

### CODE REVIEW

File: custom_components/spotcast/__init__.py [hunk 1]
```diff
@@ -302,15 +305,18 @@ def start_casting(call):
...
-            time.sleep(3)
+        if shuffle or repeat or start_volume_pct <= 100:
+            if start_volume_pct <= 100:
+                _LOGGER.debug("Setting volume to %d", start_volume_pct)
+                time.sleep(2)
+                client.volume(volume_percent=start_volume_pct, device_id=spotify_device_id)
             if shuffle:
                 _LOGGER.debug("Turning shuffle on")
-                time.sleep(2)
+                time.sleep(5)
```
Comment (fondberg) [inline custom_components/spotcast/__init__.py:315]: Why 5?

**Review by fondberg — COMMENTED** [review]
Thanks for your PR. Have one small question

File: custom_components/spotcast/__init__.py (hunk 1 above)
Comment (justincrosby) [reply custom_components/spotcast/__init__.py:315]: I removed the initial sleep(3) and added it here to the sleep(2). Reason being that the volume adjustment needs to take place sooner to affect the initial output. I'm not entirely sure that it matters to wait that long to set these but it also these aren't as time sensitive.

Comment (fondberg) [reply custom_components/spotcast/__init__.py:315]: Right, I think it is ok with 3 seconds in that case in both of the cases

**Comment by fondberg** [comment]
I changed the naming and removed _pct.

**Comment by jazzyisj** [comment]
Great feature fellas!  Just a reminder to update the readme.  I had to dive into the code to get the parameter name.

**Pull request merged**